nixos/roles/webproxy.nix

```diff
@@ -37,13 +39,6 @@ in
     (lib.mkIf fccfg.enable {
       warnings = lib.optional (varnishCfg != null) "Configuring varnish via /etc/local/varnish/default.vcl is deprecated, please migrate your Configuration to Nix";
 
-      assertions = [{
-        assertion = !((config.flyingcircus.services.varnish.virtualHosts != {}) && (varnishCfg != null));
```

Comment:
You could change that assertion to varnishCfg != and virtualHosts having more than 1 item.

Reply:
the single item might be overriden forcefully, but at that point the assertion isn't effective anyways. will add